src/Mvc/Extensions.ApiDescription.Client/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -1,5 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
 <Project>
+  <ItemGroup>
```

Comment:
@dougbu  Will this cause the schema file to be imported twice? Are both of these targets files imported?

Reply:
No double import. The `build` targets file is imported only in inner builds i.e. when `TargetFrameworks` isn't set and the `buildMultiTargeting` one is imported only in outer builds. Basically the duplication ensures the items show up no matter what &hellip but only once 😄